core/domain/suggestion_services.py

```diff
@@ -63,7 +63,7 @@ def create_suggestion(
     if target_type == suggestion_models.TARGET_TYPE_EXPLORATION:
         thread_id = feedback_services.create_thread(
             target_id, None, author_id, description,
-            DEFAULT_SUGGESTION_THREAD_SUBJECT)
+            DEFAULT_SUGGESTION_THREAD_SUBJECT, True)
```

Comment:
If an arg has a default value, the name of the arg should be specified in caller functions. /cc @apb7 @kevinlee12

Reply:
Done